nrpy/examples/seobnrv5_aligned_spin_inspiral.py

```diff
@@ -47,6 +47,18 @@
 #########################################################
 # STEP 2: Declare core C functions & register each to
 #         cfc.CFunction_dict["function_name"]
+
+# Flag to compute the SEOBNR waveform in the frequency domain.
+# Ensure that the flag is the same as in line 131 of nrpy/infrastructure/BHaH/seobnr/SEOBNR_C_codegen_library.py
+APPLY_FD = False
```

Comment:
For consistency with other NRPy codes, please do not all-caps parameters. Also, pass them to the function as 
function(apply_fd = apply_fd)
so it is clear to the user that the name of the parameter in the function matches the parameter being passed.

Reply:
Also, let's work to make the parameter names less vague than "perform_iterative_refinement", "print_waveform",  "apply_fd".